### ISSUE TITLE
"Synchrnoziation Failed"

### ISSUE THREAD

**Issue by keenahn** [todotxt/todo.txt-android#481]
This just started in the last day. Tried disconnecting and reconnecting dropbox, still no luck. Anyone else getting this?

**Comment by keenahn** [comment]
I imagine it's because Dropbox has completely removed their v1 API endpoints on 9/28

https://blogs.dropbox.com/developers/2017/06/updated-api-v1-deprecation-timeline/

**Comment by torreyn** [comment]
Bump. Happened to me too.

"Failed to load folder list from Dropbox"
![2017-09-28 21 12 23](https://user-images.githubusercontent.com/665802/31000632-eb0f6828-a491-11e7-811a-ac69d057d5dc.png)

**Comment by alister** [comment]
see: #469

**Comment by mpcjanssen** [comment]
Dropbox did not only deprecate their v1 API, they have also shut it down. Expect a lot of Dropbox syncing apps to fail today.

**Comment by leofontenelle** [comment]
@keenahn, I don't know if you have the permissions to do this, but I suggest you to close this bug report as a duplicate of #469.

**Comment by wjdodd** [comment]
I'm getting the same behavior, even after reinstalling the app. I just noticed it today; everything worked wonderfully before this. I hope the app can be updated so it's usable again.

**Comment by rlpowell** [comment]
People might want to be aware of the existence of Simpletask, which is a very similar app.  https://play.google.com/store/apps/details?id=nl.mpcjanssen.todotxtholo

**Comment by mathewjowens** [comment]
I just took a look at Simpletask, as todo.txt stopped working. Same issue for me - can't point it at the correct file location within Dropbox.

**Comment by afischl** [comment]
> I just took a look at Simpletask, as todo.txt stopped working. Same issue for me - can't point it at the correct file location within Dropbox.

@mathewjowens You've most likely got such a directory after allowing Simpletask to access DropBox:  [www.dropbox.com/home/Apps/Simpletask App Folder](https://www.dropbox.com/home/Apps/Simpletask%20App%20Folder) - just move your _todo.txt_ file in there (e.g. with the DropBox website or your synched folder if you have the DropBox client installed on your PC).

**Comment by mathewjowens** [comment]
Hmm, nope. I approved Dropbox, but no folder created (and no ability to navigate within Dropbox file structure). 

But I can get Simpletask working with FolderSync. I just need to remember to sync before I use the app.

**Comment by karbassi** [comment]
Also related to todotxt/todo.txt-ios#256

**Comment by stoffe123** [comment]
Are there any news on this? The bug makes the app useless and it's an app that many people paid money for it would be great to get some information at least of a time plan of this since what causing the bug is a known and foreseeable api change in dropbox.

If there is no feedback I guess you can claim your money back for the app?

**Comment by Surf2Sun** [comment]
Does anyone know the todo.txt file location on the phone so I can manually sync it?

**Comment by ginatrapani** [comment]
Hi all -- I just posted this to the blog and the mailing list regarding this issue: http://blog.todotxt.org/post/166157430787/todotxt-for-android-and-ios-stopped-syncing-to

**Comment by boydthomson** [comment]
I've been so happy with this app for years.  It has been well worth my $2.
Surf2Sun had a good idea of using another app (such as DropSync) to handle the syncing.  If the todo.txt app saves the text file somewhere in the file system that would be a great workaround for me.  I've poked around but haven't been able to find a location with a todo.txt on my phone.  Perhaps someone with more skilz could suggest where/how to look...?

**Comment by stoffe123** [comment]
Thanks for the feedback and info although it doen't make sense to me that an app that must have brought in thousands of dollars can't be maintained at the state that it at least is possible to build.

**Comment by ginatrapani** [comment]
@stoffe123 Yep, I understand, and I wish the apps hadn't fallen so far into the state of disrepair that they did. I have no other explanation other than that I'm not an Android or iOS developer in my professional life, and things in my personal life changed enough over the years that I no longer had the nights and weekend time to keep those skills up-to-date in order to maintain them myself. The apps bring in 10s of dollars per month, not enough to pay another developer anywhere near a decent rate to work on them, and the volunteer participation wasn't high enough to keep the work going. I take partial responsibility here for the pull requests that didn't get attention over the years as well. The move to a community-maintained org removes me as the sole blocker from here on in, but again, finding volunteers with both the time and the skills to make these updates isn't easy. It will continue to be a challenge.

**Comment by penpendede** [comment]
It is a pity that I am not into native apps. Otherwise I'd try to fix the pressing issue of dropbox support as todo.txt is one of the few apps I truly love to use :-(

**Comment by beavel** [comment]
Started work on this today, and have made some progress. This is my learn a few things about Java project, so veteran eyes are welcome. I haven't programmed in Java before, but really wanted to fix the app.

**Comment by beavel** [comment]
I have a working version of the updated app with v2 SDK. Please read the PR details for tested functionality and things still needing decisions. PR #508 can be ignored if PR #509 is taken as I continued the changes off of 508. 

@karbassi & @evanp Let me know if you have any questions or issues as you appear to be the active maintainers ATM.
